MANIFEST.in

```diff
@@ -0,0 +1 @@
+graft src/dartsort/pretrained
```

Comment:
thanks for this. lead me here: https://stackoverflow.com/a/14159430/3979938 . who knew! i had the package-data part in my pyproject.toml but not this thing